app/models/service_template_transformation_plan_task.rb

```diff
@@ -129,4 +201,60 @@ def create_error_status_task(userid, msg)
       :message => msg
     )
   end
+
+  def conversion_options_source_provider_vmwarews_vddk(_storage)
+    {
+      :vm_name            => source.name,
+      :transport_method   => 'vddk',
+      :vmware_fingerprint => source.host.thumbprint_sha1,
```

Comment:
And you should assert at some point that `source.host.kind_of?(ManageIQ::Providers::Vmware::InfraManager::HostEsx)`

Reply:
Well if we add support for some other source provider type in the future that isn't vmware?  I guess it would never get to this method since it is already based on the emstype right.  In that case I'm good with it, didn't notice this was already in a `vmwarews` method.